.changeset/dry-waves-remain.md

```diff
@@ -0,0 +1,27 @@
+---
+'@graphql-inspector/github-loader': patch
```

Comment:
I see dependencies updated in `github`, `github-loader` and `url-loader`, why do you need to bump everything?